db/db_impl_files.cc

```diff
@@ -350,6 +242,8 @@ bool CompareCandidateFile(const JobContext::CandidateFileInfo& first,
 };  // namespace
 
 // Delete obsolete files and log status and information of file deletion
+// Note: All WAL files must be deleted through this function (unelss they are
```

Comment:
Is this note necessary? Seems to be a left-over from the previously reverted commit.